2020/4/passport/src/main.rs

```diff
@@ -0,0 +1,248 @@
+use std::env;
+use std::fs::File;
+use std::io::{self, BufRead};
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    let file = File::open(&args[1]);
+
+    let reader = io::BufReader::new(file.unwrap());
+    let mut current = String::from("");
+    let mut passports = Vec::new();
+    for line in reader.lines() {
+        let l = line.unwrap();
+        if l.len() == 0 {
+            let passport = Passport::from_string(&current);
+            passports.push(passport);
+            current = String::from("");
```

Comment:
I've often used `"".to_string()` but I really don't know which is idiomatic (if any). I think I actually prefer your pattern anyway.